packages/backend/config/config.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
**issue (blocking):**  This file should not contain hard-coded db details and should support dynamic variables from an env file based on each users machine